Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,8 @@
 
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
 
+    <uses-permission android:name="android.permission.WRITE_SETTINGS" tools:ignore="ProtectedPermissions" />
+
```

Comment:
this won't be necessary, please see my suggestion below in `QrCodeFullScreenFragment` file ?